flink-container/docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Apache Flink cluster deployment on docker using docker-compose
+
+## Installation
+
+Install the most recent stable version of docker
+https://docs.docker.com/installation/
+
+## Build
+
+Images are based on the official Java Alpine (OpenJDK 8) image. If you want to
+build the flink image run:
+
+    sh build.sh --job-jar /path/to/job/jar/job.jar --image-name flink:job
+
+or
+
+    docker build -t flink .
```

Comment:
There are some args missing to make these commands work. Is it intentional?

Reply:
No not really. I will update the documentation. Thanks for catching it.